mkdocs.yaml

```diff
@@ -49,6 +48,10 @@ plugins:
   - mkdocs-jupyter
   # https://mkdocstrings.github.io/
   - mkdocstrings:
+      handlers:
```

Comment:
What is this for?

Reply:
It includes the submodules automatically, e. g. bofire.data_models.features.categorical, bofire.data_models.features.continuous etc.
It should not be used on the top level bofire or bofire.data_models though, as it will put too much on one page.
Instead we may want to add a few more ref.md files